config/variables.js

```diff
@@ -199,8 +199,21 @@ const variables = {
     'screen-tablet': 768,
     'screen-sm-mobile': 400
   },
+  dimensions: {
+    'grid-base': '8px',
+    'spacing-base': '24px',
+    'homepage-max-width': '800px',
+    'site-max-width': '1200px',
+    'site-margins': '24px',
+    'site-margins-mobile': '16px',
+    'site-min-width': '320px'
+  },
```

Comment:
I mentioned in the previous PR we don't need this object. Grid is already defined at the root level and spacing base should be calculated based on that. The other variables are old BSD ACCESS variables.